week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,12 @@
 function getNumber() {
-    return Math.random() * 10;
+	return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? return a random number from between 0-9(inclusive)
```

Comment:
Very almost right (and closer than most people got)!

It's correct that it will never return 10 exactly, but because it does not only return whole (integer) numbers, it can also return `9.1` or `9.99999` or any other decimal between 9 and 10.